set.mm

```diff
@@ -632529,6 +632529,118 @@ fixed reference functional determined by this vector (corresponding to
+      PZQZBCULUMUOBOCPZQZFGHAUPFRPUNISUOGRPUPJUAAUPUNQZQZHRPZTUQUTTKUSUQUTUSAUN
+      UPQZQZUQURVAAUPUNUBUCUQVBAUNUPUFUDUGUEUHUIUJUK $.
+  $}
+
+  ${
+    $d A x $.  $d B k x $.  $d G x $.  $d k ph x $.
+    3factsumint3.1 $e |- A = ( L [,] U ) $.
+    3factsumint3.2 $e |- ( ph -> L e. RR ) $.
+    3factsumint3.3 $e |- ( ph -> U e. RR ) $.
+    3factsumint3.4 $e |- ( ( ph /\ x e. A ) -> F e. CC ) $.
+    3factsumint3.5 $e |- ( ph -> ( x e. A |-> F ) e. ( A -cn-> CC ) ) $.
+    3factsumint3.6 $e |- ( ( ph /\ k e. B ) -> G e. CC ) $.
+    3factsumint3.7 $e |- ( ( ph /\ ( x e. A /\ k e. B ) ) -> H e. CC ) $.
+    3factsumint3.8 $e |- ( ( ph /\ k e. B ) -> ( x e. A |-> H ) e.
+     ( A -cn-> CC ) ) $.
+    $( Lemma for first equation in lcm inequality proof.  (Contributed by
+       metakunt, 26-Apr-2024.) $)
+    3factsumint3 $p  |- ( ph -> sum_ k e. B S. A ( G x. ( F x. H ) ) _d x
+    = sum_ k e. B ( G x. S. A ( F x. H ) _d x ) ) $=
+      ( co wcel cmul cv wa cc adantlr wi ancom anbi2i anass bicomi bitri imbi1i
+      citg mpbi mulcld cr cmpt cicc ccncf cibl adantr mulcncf oveq1i cnicciblnc
+      eleqtrdi syl3anc itgmulc2 eqcomd sumeq2dv ) ADBCHGIUASZUASUMZHBCVJUMUASZF
+      AFUBDTZUCZVLVKVNBCVJHUDPVNBUBCTZUCZGIAVOGUDTVMNUEAVOVMUCZUCZIUDTZUFVPVSUF
+      QVRVPVSVRAVMVOUCZUCZVPVQVTAVOVMUGUHVPWAAVMVOUIUJUKULUNUOVNJUPTZEUPTZBCVJU
+      QZJEURSZUDUSSZTWDUTTAWBVMLVAAWCVMMVAVNWDCUDUSSZWFVNBGICABCGUQWGTVMOVARVBC
+      WEUDUSKVCVEJEWDVDVFVGVHVI $.
+  $}
+
+  ${
+    $d A k $.  $d B k $.  $d F k $.  $d k ph x $.
+    3factsumint4.1 $e |- ( ph -> B e. Fin ) $.
+    3factsumint4.2 $e |- ( ( ph /\ x e. A ) -> F e. CC ) $.
+    3factsumint4.3 $e |- ( ( ph /\ k e. B ) -> G e. CC ) $.
+    3factsumint4.4 $e |- ( ( ph /\ ( x e. A /\ k e. B ) ) -> H e. CC ) $.
+    $( Lemma for first equation in lcm inequality proof.  (Contributed by
+       metakunt, 26-Apr-2024.) $)
+    3factsumint4 $p |- ( ph -> S. A sum_ k e. B ( F x. ( G x. H ) ) _d x =
+       S. A ( F x. sum_ k e. B ( G x. H ) ) _d x ) $=
+      ( cmul co csu cv wcel wa cc wi cfn adantr anass bicomi imbi1i mpbi mulcld
+      adantlr fsummulc2 eqcomd itgeq2dv ) ABCDFGHMNZMNEOZFDULEOMNZABPCQZRZUNUMU
+      PDULFEADUAQUOIUBJUPEPDQZRZGHAUQGSQUOKUHAUOUQRRZHSQZTURUTTLUSURUTURUSAUOUQ
+      UCUDUEUFUGUIUJUK $.
+  $}
+
+  ${
+    $d A k x $.  $d B k x $.  $d F k $.  $d G x $.  $d k ph x $.
+    3factsumint.1 $e |- A = ( L [,] U ) $.
+    3factsumint.2 $e |- ( ph -> B e. Fin ) $.
+    3factsumint.3 $e |- ( ph -> L e. RR ) $.
+    3factsumint.4 $e |- ( ph -> U e. RR ) $.
+    3factsumint.5 $e |- ( ph -> ( x e. A |-> F ) e. ( A -cn-> CC ) ) $.
+    3factsumint.6 $e |- ( ( ph /\ k e. B ) -> G e. CC ) $.
+    3factsumint.7 $e |- ( ( ph /\ k e. B ) -> ( x e. A |-> H ) e.
+     ( A -cn-> CC ) ) $.
+    $( First equation in lcm inequality proof.  (Contributed by metakunt,
+       26-Apr-2024.) $)
+    3factsumint $p  |- ( ph -> S. A ( F x. sum_ k e. B ( G x. H ) ) _d x =
```

Comment:
What is the meaning of this equation? It should be described in more detail (especially since the  "lcm inequality proof" is not available yet).

Reply:
I have added documentation outlining the idea behind this theorem

Reply:
The link seems to be missing a tilde ~ : [https://www3.nd.edu/~andyp/notes/AKS.pdf](https://www3.nd.edu/~andyp/notes/AKS.pdf)

Reply:
I added the tilde back, but after rewrapping it just looks wrong. I don't know how metamath.exe handles tildes in links

Reply:
I might have to change tilde to %7e, don't I?

Reply:
%7e works, at least

Reply:
To be honest, I did not find this theorem in the referenced documentation. 
But maybe everything becomes clearer after more theorems are available for this topic. 

Since this is currently in a mathbox only, I will accept PR.

Reply:
I saw the equations on which @metakunt is working in the proof of theorem 3.1.